_sass/_objects--notify-form.scss

```diff
@@ -0,0 +1,18 @@
+$size--input-height:                          45px;
+$size--input-width:                           305px;
+$color--input-border:                         $alto;
+
+.notify-form {
```

Comment:
I like it `contact` and within a `notify-form` 👍 , HOWEVER, maybe the `contact` will be more like a `layout` or even `partial`? I mean you define positions and special styles for that section and whenever the section is displayed will shape its children like that